python/graphscope/interactive/__init__.py

```diff
@@ -18,8 +18,12 @@
 
 # The gremlinpython has a async event loop, which may conflicts with
 # jupyter notebook's event loop.
-import nest_asyncio
 
-nest_asyncio.apply()
+try:
+    import nest_asyncio
+
+    nest_asyncio.apply()
+except:  # noqa: E722
+    pass
```

Comment:
Why skip? I remember that we have already installed `nest_asyncio` into our dependencies in `requirements.txt`.